src/planwise/client/projects/handlers.cljs

```diff
@@ -22,8 +22,10 @@
 (register-handler
  :projects/invalidate-projects
  in-projects
- (fn [db [_]]
-   (update db :list asdf/invalidate!)))
+ (fn [db [_ new-projects]]
+   (if new-projects
+     (update db :list asdf/invalidate! into new-projects)
```

Comment:
Actually it is now used in `current_project/handlers` since e0027ef8b552bafa9542c26480e2f89341b73f4a. 

Though I'm not a fan of this branch either, doing the invalidate-optimistically-update in-place would imply directly modifying the projects list from the `current_project` handlers, this is, outside the `in-current-project` path.